### PR TITLE
Handle skipped backend selection as no backend if not already paired

### PR DESCRIPTION
**Pull request by NeonDaniel**

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -735,7 +737,7 @@ def handle_personal_backend_url(self, message):
         self.state = SetupState.PAIRING
         self.pairing.kickoff_pairing()
 
-    def handle_no_backend_selected(self, message):
+    def handle_no_backend_selected(self, _):
```
Comment (JarbasAl) [inline __init__.py:740]: why this change? please keep keyword argument, this helps in dig_for_message etc, for consistency we always want the message object to be explicit, not "_"

at any point that a message is received, we need to allow .forward/.reply

File: __init__.py [hunk 2]
```diff
@@ -593,6 +593,8 @@ def handle_language_back_event(self, message):
     @killable_event(msg="pairing.backend.menu.stop")
     def handle_backend_menu(self):
         if not self.settings["enable_backend_selection"]:
+            if not is_paired():
+                self.handle_no_backend_selected(None)
             if self.settings["enable_stt_selection"]:
```
Comment (JarbasAl) [inline __init__.py:598]: `elif`

File: __init__.py [hunk 3]
```diff
@@ -593,6 +593,8 @@ def handle_language_back_event(self, message):
     @killable_event(msg="pairing.backend.menu.stop")
     def handle_backend_menu(self):
         if not self.settings["enable_backend_selection"]:
+            if not is_paired():
+                self.handle_no_backend_selected(None)
```
Comment (JarbasAl) [inline __init__.py:597]: this will need an extra check in this handler also, it calls STT unconditionally 

```
        # continue to STT
        self.handle_stt_menu()
 ```

File: __init__.py (hunk 1 above)
Comment (NeonDaniel) [reply __init__.py:740]: Just resolving editor warnings, I'll change it back

File: __init__.py (hunk 3 above)
Comment (NeonDaniel) [reply __init__.py:597]: There's a check in `handle_stt_menu()`; maybe we can avoid checking twice though..
